app/views/search/banners/_rdv_solidarites.html.slim

```diff
@@ -1,11 +1,10 @@
 - if context.invitation?
-  | Vous avez été invité(e) à prendre&nbsp;
-  span rendez-vous
+  | J'organise mon rendez-vous
```

Comment:
On essaye de faire plus attention au design, et d'avoir un ton (ou "tone and voice" comme disent les design system anglophones) cohérent à travers l'appli. Ça veut dire que le changement de la deuxième à la première personne n'est pas anodin, et surtout on n'a pas du tout envie de se lancer là dedans.

Les actions faites par l'utilisateur sont soit à la deuxième personne quand on lui donne des consignes ("Choisissez un créneau"), soit à l'infinitif quand c'est des boutons qui proposent des actions ("Enregistrer").
Ici, si c'est le fait que c'est pas forcément le département qui envoie l'invitation, on pourrait tout simplement enlever l'info qui n'est plus pertinente : 
```suggestion
  | Vous avez été invité·e à prendre rendez-vous
```
(et c'est vrai que les parenthèses c'est moche, un point médian c'est un peu plus joli)